JenniferLemos-tarefaS05/tarefacasas05.py

```diff
@@ -27,107 +30,129 @@ def obter_opcao():
+            nome, sobrenome = nome_sobrenome.split(' ')
+            break
         except ValueError:
-            print("Entrada inválida. Por favor, digite um número válido ou 'fim' para terminar.")
-
-    if len(notas) == 0:
-        print("Nenhuma nota válida foi adicionada. Aluna não incluída.")
-        return
-
-    dataset['alunas'].append({
-        'nome': nome,
-        'faltas': faltas,
-        'notas': notas
-    })
-    print("Aluna incluída com sucesso!")
-
-def consultar_lista_alunas(dataset):
-    if len(dataset['alunas']) == 0:
-        print("\nNão há registros de alunas cadastradas.")
-        return
-
-    print("\nLista de Alunas:")
-    for aluna in dataset['alunas']:
-        print(f"Nome: {aluna['nome']}, Faltas: {aluna['faltas']}, Notas: {aluna['notas']}")
-
-def consultar_faltas_aluna(dataset):
-    if len(dataset['alunas']) == 0:
-        print("\nNão há registros de alunas cadastradas.")
-        return
-
-    nome = input("Digite o nome da aluna: ")
-    for aluna in dataset['alunas']:
-        if aluna['nome'] == nome:
-            print(f"Aluna: {aluna['nome']}, Faltas: {aluna['faltas']}")
-            return
-    print("Aluna não encontrada.")
-
-def consultar_notas_aluna(dataset):
-    if len(dataset['alunas']) == 0:
-        print("\nNão há registros de alunas cadastradas.")
-        return
-
-    nome = input("Digite o nome da aluna: ")
-    for aluna in dataset['alunas']:
-        if aluna['nome'] == nome:
-            print(f"Aluna: {aluna['nome']}, Notas: {aluna['notas']}")
-            return
-    print("Aluna não encontrada.")
-
-def consultar_status_aprovacao(dataset):
-    if len(dataset['alunas']) == 0:
-        print("\nNão há registros de alunas cadastradas.")
-        return
-
-    nome = input("Digite o nome da aluna: ")
-    for aluna in dataset['alunas']:
-        if aluna['nome'] == nome:
-            media = sum(aluna['notas']) / len(aluna['notas'])
-            presenca = 1 - (aluna['faltas'] / 100)  
-            if media >= 6 and presenca >= 0.8:  ## Calcula a presença / 80 porcentagem para aprovação.
-                status = "Aprovada"
-            else:
-                status = "Reprovada"
-            print(f"Aluna: {aluna['nome']}, Média: {media:.2f}, Status: {status}")
-            return
-    print("Aluna não encontrada.")
+            print("Digite um NOME e um SOBRENOME separado por espaço")
+    return nome, sobrenome
 
-main()
+def notas_aluna():
+    quantidade_notas = int(input("Qual a quantidade de notas?: "))
+    notas = []
 
+    for quantidade in range(quantidade_notas):
+        while True:
+            try:
+                nota = input(f"Insira a {quantidade + 1}ª nota:")
+                nota = float(nota.replace(',', '.'))
+                notas.append(nota)
+                break
+            except ValueError:
+                print("Entrada inválida. Por favor, insira um número válido.")
+    return notas
+
+def presenca_das_alunas():
```

Comment:
Assim como em notas_aluna, o nome da função deve refletir a funcionalidade desta função.

**Sugestão:**
```suggestion
def obter_presenca_das_alunas():
```